src/types.rs

```diff
@@ -15,6 +17,12 @@ pub struct Coordinate<T>
     pub y: T,
 }
 
+impl<T: Float> From<(T, T)> for Coordinate<T> {
```

Comment:
Related: #32
I'm not sure where I stand on this. On the one hand, I'm personally in favour of coming down firmly on one side of the consensus, clearly stating that it's `x, y`, and letting libraries that want to use georust deal with that (which is what the ecosystem in general has to do now). On the other hand, I realise that's just my personal preference.